reference/cookie_consent.yml

```diff
@@ -0,0 +1,94 @@
+openapi: 3.0.1
+servers:
+  - url: 'https://{$$.env.store_domain}/api/storefront'
+info:
+  title: Storefront Cookie Consent API
+  description: Storefront API for specifying consent to particular categories of cookies for compliance purposes.
+  version: Storefront
+tags:
+  - name: Customers
```

Comment:
Is "Customers" the correct tag to use for this API? "Cookie Consent" seems to be more appropriate. Can you please verify?

Reply:
Take should be `Consent`.